GameDataParser/Parsers/ShopParser.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using GameDataParser.Files;
+using Maple2Storage.Types;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    public class ShopParser : Exporter<List<ShopMetadata>>
+    {
+        public ShopParser() : base(null, "shop") { }
+
+        protected override List<ShopMetadata> Parse()
+        {
+            List<ShopMetadata> shops = new List<ShopMetadata>();
+
+            /*
+             * Shop Name: Rumi
+             * Map: Lith Harbor
+             */
+            ShopMetadata rumi = new ShopMetadata()
+            {
+                TemplateId = 11000079,
+                Id = 103,
+                Category = 3,
+                Name = "shop",
+                RestrictSales = false,
+                CanRestock = false,
+                NextRestock = DateTimeOffset.UtcNow.ToUnixTimeSeconds(),
+                AllowBuyback = true,
+                Items = new List<ShopItem>
+                {
+                    new ShopItem
+                    {
+                        UniqueId = 436270,
+                        ItemId = 63000000,
+                        TokenType = (byte) Currency.Meso,
```

Comment:
If I use `Currency.Meret` it shows as Guild Funds coins. Need to hardcore the values in enum `Currency.cs`